Sources/WalletConnectNotify/Client/Wallet/NotifyClient.swift

```diff
@@ -132,6 +132,11 @@ public class NotifyClient {
 
 #if targetEnvironment(simulator)
 extension NotifyClient {
+
+    public var subscriptionChangedPublisher: AnyPublisher<[NotifySubscription], Never> {
```

Comment:
didn't we want to make this one internal?

Reply:
I decided to make it public because we have `#if targetEnvironment(simulator)` check